crates/crates_io_database/src/schema.patch

```diff
@@ -45,6 +45,22 @@
          /// The `target` column of the `dependencies` table.
          ///
          /// Its SQL type is `Nullable<Varchar>`.
+@@ -536,13 +536,14 @@
+         ///
+         /// Its SQL type is `Nullable<Timestamptz>`.
+         ///
+         /// (Automatically generated by Diesel.)
+         token_generated_at -> Nullable<Timestamptz>,
+         /// Whether this email is the primary email address for the user.
+-        is_primary -> Bool,
++        #[sql_name = "is_primary"]
++        primary -> Bool,
+     }
+ }
+
+ diesel::table! {
+     /// Representation of the `follows` table.
+     ///
```

Comment:
since the patch file is already a bit hard to maintain, I think it might be easier to just use `is_primary` directly in our code? 😅